_docs/quick-start.md

```diff
@@ -55,6 +67,9 @@ We recommend that you write your TypeScript in [VS Code](https://code.visualstud
 		"noLib": true,
 		"strict": true,
 		"target": "es6",
+
+		// required, semi-configurable
+		// (must end in "node_modules/@rbxts")
```

Comment:
I think it's okay to remove this line. We properly describe it below.

Reply:
Noted. Will remove.